lib/tasks/complete_technical_failure_emails.rake

```diff
@@ -0,0 +1,9 @@
+namespace :complete_technical_failure_emails do
+  desc "Update emails marked as technical failure to have a completed_at time"
+  task fill_completed_at_time: :environment do
+    incomplete_technical_failure_emails = Email.where(failure_reason: "technical_failure").where(finished_sending_at: nil)
+    incomplete_technical_failure_emails.each do |email|
```

Comment:
```suggestion
    incomplete_technical_failure_emails.find_each do |email|
```

This avoids needing to instance an email class each time and instead can re-use the same instance, so it'll be more efficient.

https://apidock.com/rails/ActiveRecord/Batches/find_each